### ISSUE TITLE
Update to latest Fluid to resolve circular dependency warnings in node@14

### ISSUE THREAD

**Issue by the-t-in-rtf** [fluid-project/fluid-lint-all#36]
Currently, using this package (for example, using `npm run lint`) throws warnings like the following:

```
(node:5218) Warning: Accessing non-existent property 'module' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
(node:5218) Warning: Accessing non-existent property 'module' of module exports inside circular dependency
```

**Comment by the-t-in-rtf** [comment]
Updating our dependency reveals errors coming from other packages:

```
(node:5218) Warning: Accessing non-existent property 'module' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
(node:5218) Warning: Accessing non-existent property 'module' of module exports inside circular dependency
```

```
(node:6180) Warning: Accessing non-existent property 'module' of module exports inside circular dependency
    at emitCircularRequireWarning (node:internal/modules/cjs/loader:698:11)
    at Object.get (node:internal/modules/cjs/loader:712:5)
    at Object.<anonymous> (/Users/duhrer/Source/rtf/fluid-lint-all/node_modules/markdownlint-config-fluid/node_modules/infusion/src/module/fluid.js:181:24)
```

If I manually hack their version and reinstall, the warnings disappear. I'll create pulls for each shortly.

**Comment by greatislander** [comment]
@the-t-in-rtf I've just published markdownlint-config-fluid@0.1.3 with your fix.
